src/main/java/net/dv8tion/jda/internal/entities/MemberImpl.java

```diff
@@ -116,6 +116,12 @@ public OffsetDateTime getTimeBoosted()
         return boostDate != 0 ? Helpers.toOffset(boostDate) : null;
     }
 
+    @Override
+    public boolean isBoosting()
+    {
+        return boostDate != 0;
```

Comment:
This is perhaps out of scope for this PR but why is a that variable called `boostDate` when it is a long? I think it contains a timestamp so `boostTimestamp` would fit more and is less confusing to read.

Reply:
Well... yeah, but I'm not sure if I should change it. It's private and internal anyways, but then we should also change `joinDate`, which is a long.

Reply:
IMO it would best fit in another PR, since it would change a lot of the method names and usages.